src/backend/logger/logfile.ts

```diff
@@ -87,9 +87,9 @@ export function createNewLogFileAndClearOldOnes(): createLogFileReturn {
       })
         .filter((dirent) => dirent.isFile())
         .map((dirent) => dirent.name)
-        .filter((filename) => !keep.includes(`${logDir}/${filename}`))
+        .filter((filename) => !keep.includes(`${logDir}${path.sep}${filename}`))
```

Comment:
that could be also just `join(logDir, filename)`

Reply:
updated :+1: